ui/lib/apps/SearchLogs/components/SearchResult.tsx

```diff
@@ -1,9 +1,6 @@
-import client from '@pingcap-incubator/dashboard_client'
-import {
-  LogsearchSearchTarget,
-  LogsearchTaskModel,
-} from '@pingcap-incubator/dashboard_client'
-import { Card } from '@pingcap-incubator/dashboard_components'
+import client from '@lib/client'
+import { LogsearchSearchTarget, LogsearchTaskModel } from '@lib/client'
```

Comment:
mark

Reply:
yeah, I did a global search & replace, so there will be some duplicate imports..